### ISSUE TITLE
Conda issue installing on EC2

### ISSUE THREAD

**Issue by shepherddev1** [bcbio/bcbio-nextgen#2480]
Hello, I'm having an issue installing on EC2. Anaconda2 is installed and things appear to proceed correctly until this step: 

~/install/bcbio-vm/anaconda/bin/conda install --yes -c bioconda bcbio-nextgen-vm
Solving environment: failed

PackagesNotFoundError: The following packages are not available from current channels:

  - bcbio-nextgen-vm
  - elasticluster
  - ansible
  - httplib2
  - bcbio-nextgen-vm
  - elasticluster
  - oauth2client
  - bcbio-nextgen-vm
  - elasticluster
  - google-api-python-client
  - bcbio-nextgen-vm
  - arvados-cwl-runner
  - arvados-python-client
  - bcbio-nextgen-vm
  - arvados-cwl-runner
  - schema-salad[version='>=2.6.20171116190026']
  - ruamel.yaml[version='>=0.12.4,<0.15']
  - bcbio-nextgen-vm
  - arvados-cwl-runner
  - schema-salad[version='>=2.6.20171116190026']
  - typing[version='>=3.5.3,<3.6']
  - bcbio-nextgen-vm
  - dxpy
  - python-magic[version='>=0.4.6']
  - bcbio-nextgen-vm
  - dxpy
  - xattr[version='>=0.6.4']
  - bcbio-nextgen-vm
  - dx-cwl[version='>=0.1.0a20171222']
  - bcbio-nextgen-vm
  - pyhocon

Current channels:

  - https://conda.anaconda.org/bioconda/linux-64
  - https://conda.anaconda.org/bioconda/noarch
  - https://repo.anaconda.com/pkgs/main/linux-64
  - https://repo.anaconda.com/pkgs/main/noarch
  - https://repo.anaconda.com/pkgs/free/linux-64
  - https://repo.anaconda.com/pkgs/free/noarch
  - https://repo.anaconda.com/pkgs/r/linux-64
  - https://repo.anaconda.com/pkgs/r/noarch
  - https://repo.anaconda.com/pkgs/pro/linux-64
  - https://repo.anaconda.com/pkgs/pro/noarch

To search for alternate channels that may provide the conda package you're
looking for, navigate to

    https://anaconda.org

and use the search bar at the top of the page.

[ec2-user@ip-172-31-34-48 ~]$ ln -s ~/install/bcbio-vm/anaconda/bin/bcbio_vm.py /usr/local/bin/bcbio_vm.py

Manually installing packing with anaconda works correctly, so I'm unclear about the cause. Thank you for your help.

**Comment by chapmanb** [comment]
Sorry about the install problems. If you're doing a bcbio-vm install you also need to include conda-forge in your channels, which should resolve the issue:
```
~/install/bcbio-vm/anaconda/bin/conda install --yes -c conda-forge -c bioconda bcbio-nextgen-vm
```
Is there out of date documentation that is missing the conda-forge channel? If you could point us at it I'd be happy to fix to avoid other folks tripping on this. Thanks again for the feedback and hope this gets the install finished for you.

**Comment by chapmanb** [comment]
Thanks so much for the heads up and apologies about the out of date documentation. I've updated that to have the correct command with conda-forge included, apologies for the ommission.

I've also updated this to reflect the current state of Cloud support in bcbio. We're actively moving to use Common Workflow Language rather than this custom approach in bcbio, and right now recommend using a simplified approach where you spin up single muticore machines and manage using ansible. I added pointers to all this in the documentation.

Thank you again for the feedback and trying this out.
